### PR TITLE
Add Ticker, Hourly Ticker, Order Book, Transanction model

### PR DESCRIPTION
**Pull request by nejcgorsic**
Refactored models and models usage.
Added the following models and updated the code in example and client accordingly:
* Ticker
* Hourly ticker
* Order book
* Transanction

Added examples print.
image
image
image

### CODE REVIEW

File: rest/models.py [hunk 1]
```diff
@@ -12,3 +13,64 @@ def __init__(self, response: requests.Response):
...
+        self.volume = dict_rsp.get('volume')
+        self.low = dict_rsp.get('low')
+        self.ask = dict_rsp.get('ask')
+        self.open = dict_rsp.get('open')
+
+
+class OrderBook:
+    timestamp: str
+    microtimestamp: str
+    bids: typing.List
```
Comment (potocnikj) [inline rest/models.py]: This is incorrect type for bids and asks. Should be at least `typing.List[typing.List[[str]]`. However, each list includes 2 things.
Price and Amount. hence, I suggest better modeling.

You see, now we have issues, as someone using order book entry doesn't have much use for it. Has to loop over 2 grades of lists and .... uugh, ugly as hell.

Lets use some named tuples!
We add one import and define our named tuple at the top of this file. Something like:
```
from collections import namedtuple

OrderBookEntry = namedtuple('OrderBookEntry', 'price amount')
```

OrderBook class should then have private instance method.

```
def _serialize_order_book_entries(entries: typing.List[typing.List[str]]) -> typing.List[OrderBookEntry]:
         serialized = []
         for entry in entries:
                serialized.append(OrderBookEntry(entry[0], entry[1])

        return serialized
```
We now have modeling that we want. in constructor you now call something like:

```
self.bids = self._serialize_order_book_entries(dict_rsp.get('bids'))
self.asks = self._serialize_order_book_entries(dict_rsp.get('asks'))
```
Then someone can use this order book much easier.
Like: `order_book.bids[0].price`

Comment (potocnikj) [reply rest/models.py]: Ough, I wrote this code without trying it out, maybe some typos or so. check it out.

File: rest/models.py [hunk 2]
```diff
@@ -12,3 +13,64 @@ def __init__(self, response: requests.Response):
...
+        self.last = dict_rsp.get('last')
+        self.timestamp = dict_rsp.get('timestamp')
+        self.bid = dict_rsp.get('bid')
+        self.vwap = dict_rsp.get('vwap')
+        self.volume = dict_rsp.get('volume')
+        self.low = dict_rsp.get('low')
+        self.ask = dict_rsp.get('ask')
+        self.open = dict_rsp.get('open')
+
+class HourlyTicker:
```
Comment (potocnikj) [inline rest/models.py]: This model is completely redundant. It's completely the same as `Ticker`. It doesn't matter whether is hourly or daily or weekly, it's still ticker.
Hourly ticker should also return Ticker model.

**Review by potocnikj — CHANGES_REQUESTED** [review]
One more things: Commit messages are better now, but still not good.
They should never include commas or dots or something.

Message should be:
`Do something good in this commit` 
And not:
`Do something good in this commit.`

Also commit messages you write are too expressive. Should be more atomic. If you cant explain what it does in a single very short sentence, it probably means, that code changes should be split to more commits.

File: rest/models.py (hunk 1 above)
Comment (nejcgorsic) [reply rest/models.py]: This has been placed in case of order book and evolved.
Resolving as I see no futher conflict.

File: rest/models.py (hunk 2 above)
Comment (nejcgorsic) [reply rest/models.py]: Model removed, moved to Ticker only.
Resolving as I see no futher conflict.

**Comment by nejcgorsic** [comment]
@potocnikj @vidpotocnik 

Sorry I missed your comment about commits, I will be more careful in the future.

**Review by potocnikj — APPROVED** [review]

**Pull request merged**